workers/unity/Packages/io.improbable.gdk.core/Subscriptions/Reader.cs

```diff
@@ -35,16 +37,16 @@ public TComponent Data
             }
         }
 
-        public Authority Authority
+        public bool HasAuthority
         {
             get
             {
                 if (!IsValid)
                 {
-                    throw new InvalidOperationException("Cannot read authority when Reader is not valid");
+                    throw new InvalidOperationException($"Cannot read {nameof(HasAuthority)} when Reader is not valid");
                 }
 
-                return ComponentUpdateSystem.GetAuthority(EntityId, ComponentId);
+                return EntityManager.HasComponent(Entity, ComponentAuthType);
```

Comment:
On balance, should we make this breaking change?

Reply:
What is the breaking change here?

Reply:
Both the type and name of the property are changing. 

We could keep this backwards compatible by changing the return to:

```
return EntityManager.HasComponent(Entity, ComponentAuthType) ? Authority.Authoritative : Authority.NotAuthoritative;
```

Reply:
Right, but removing auth loss imminent doesn't mandate or require this change

Reply:
I guess I just don't see the value in keeping backwards compatibility given we'll be imposing that `Authority` is essentially nothing but true or false

Sure it's breaking but it does simplify logic for users (no more ==/!= comparisons)

Reply:
My opinion is that the bar for making a breaking change should be relatively high and be done for good reason. With this case I don't feel the rationale is strong enough, but if you two both feel otherwise, I'm not gonna die on this hill.